docs/tutorials/start-a-private-network/3keygen.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Generate Your Own Keys
+---
+
+Now that we know the fundamentals and the command line options, it's time to generate our own keys rather than using the well-known Alice and Bob keys. Each person who wants to participate in the blockchain should generate their own keys. This page explains several options for generating keys, and each participant only needs to choose one. Regardless of which option you choose, be sure to record all of the output from this section as you will need it later.
+
+## Option 1: Subkey
+Subkey is a tool the generates keys specifically designed to be used with Substrate. This is the most secure way to generate keys, and is how you should generate keys if you plan to use them in production.
+
+Begin by compiling and install the utility.
```

Comment:
```suggestion
Begin by compiling and installing the utility.
```

Reply:
I might also suggest that you be honest with the user that this might take a while to compile. Maybe 15 min or more.

Reply:
It may be slow but could be the best approach in the long term. As you said we may want to have a separate section for generating keys and link it when needed.
The UI could be changing faster than `subkey` and it adds another thing to keep up to date every time it changes.
On the other side it is easier to use the UI than `subkey` so I think that we need to find the correct balance here.